src/SprykerEco/Zed/Payone/Business/PayoneFacadeInterface.php

```diff
@@ -609,4 +611,16 @@ public function findPayoneOrderItemStatus(int $idSalesOrder, int $idSalesOrderIt
     public function executePartialCapture(
         PayonePartialOperationRequestTransfer $payonePartialOperationRequestTransfer
     ): CaptureResponseTransfer;
+
+    /**
+     * Specification:
+     * - Starts Payone session.
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\PayoneKlarnaStartSessionRequestTransfer $klarnaStartSessionRequestTransfer
+     *
+     * @return \Generated\Shared\Transfer\PayoneKlarnaSessionResponseTransfer
+     */
+    public function startKlarnaSession(PayoneKlarnaStartSessionRequestTransfer $klarnaStartSessionRequestTransfer): PayoneKlarnaSessionResponseTransfer;
```

Comment:
* Let's rename it to the `sendKlarnaSessionRequest()`.

Reply:
++

Reply:
++